handlers/handler-flowcontrol-qps/src/main/java/org/apache/servicecomb/qps/QpsControllerManager.java

```diff
@@ -210,21 +209,17 @@ public QpsControllerManager setGlobalQpsStrategy(String globalLimitKey, String g
 
   private AbstractQpsStrategy chooseStrategy(String globalConfigKey, Long limit, Long bucket,
       String strategyKey) {
-    AbstractQpsStrategy strategy;
-    switch (StrategyType.parseStrategyType(strategyKey)) {
-      case FixedWindow:
-        strategy = new FixedWindowStrategy(globalConfigKey, limit);
-        break;
-      case LeakyBucket:
-        strategy = new LeakyBucketStrategy(globalConfigKey, limit);
-        break;
-      case TokenBucket:
-        strategy = new TokenBucketStrategy(globalConfigKey, limit, bucket);
-        break;
-      case SlidingWindow:
-      default:
-        strategy = new FixedWindowStrategy(globalConfigKey, limit);
+    AbstractQpsStrategy strategy = new FixedWindowStrategy(globalConfigKey, limit);
+    List<AbstractQpsStrategy> customStrategy = SPIServiceUtils
+        .getOrLoadSortedService(AbstractQpsStrategy.class);
+    for (AbstractQpsStrategy abstractQpsStrategy : customStrategy) {
+      if (abstractQpsStrategy.name().equals(strategyKey)) {
```

Comment:
strategyKey  better to rename to 'strategy', it is not a key , but a value.

Reply:
already has local value 'strategy'